opentelemetry-stdout/src/metrics/exporter.rs

```diff
@@ -0,0 +1,174 @@
+use core::fmt;
+use std::{
+    io::{stdout, Write},
+    sync::Mutex,
+};
+
+use async_trait::async_trait;
+use opentelemetry_api::metrics::{MetricsError, Result};
+use opentelemetry_sdk::metrics::{
+    data,
+    exporter::PushMetricsExporter,
+    reader::{
+        AggregationSelector, DefaultAggregationSelector, DefaultTemporalitySelector,
+        TemporalitySelector,
+    },
+    Aggregation, InstrumentKind,
+};
+
+use crate::MetricsData;
+
+type Encoder = Box<dyn Fn(&mut dyn Write, MetricsData) -> Result<()> + Send + Sync>;
+
+/// An OpenTelemetry exporter that writes to stdout on export.
+pub struct MetricsExporter {
```

Comment:
does the stdout exporter for Metrics have a way to make sure the periodic metric reader will be using the 10,000 msec interval instead of the 60,000 msec interval?

The spec recommends 10,000 for stdout exporter here:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/stdout.md

Reply:
Can consider this in a follow up as the reader config would have to be specified outside of the context of the exporter.